src/vma/event/event_handler_manager.cpp

```diff
@@ -407,9 +407,10 @@ void event_handler_manager::update_epfd(int fd, int operation, int events)
 	ev.events = events;
 	ev.data.fd = fd;
 	BULLSEYE_EXCLUDE_BLOCK_START
-	if (orig_os_api.epoll_ctl(m_epfd, operation, fd, &ev) < 0) {
+	if ((orig_os_api.epoll_ctl(m_epfd, operation, fd, &ev) < 0) &&
+		(!(errno == ENOENT || errno == EBADF))) {
```

Comment:
What is the reason for this change ? we should resolve the problem not modify the error message

Reply:
we should suppress the error in priv_unregister_ibverbs_events and no in update_epfd

Reply:
The same technique is used as for identical issue at https://github.com/Mellanox/libvma/blob/master/src/vma/dev/net_device_table_mgr.cpp#L455
https://github.com/Mellanox/libvma/blob/master/src/vma/iomux/epfd_info.cpp#L704